README.md

```diff
@@ -1,28 +1,34 @@
-[![Join the chat at https://gitter.im/zlsa/atc](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zlsa/atc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Travis Production Build State](https://img.shields.io/travis/openscope/openscope/master.svg)](https://github.com/openscope/openscope/tree/master)
+[![openScope Current Release](https://img.shields.io/github/release/openscope/openscope.svg)](https://github.com/openscope/openscope/releases)
+[![License: MIT](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE.md)
+[![Slack Status](http://slack.openscope.co/badge.svg)](http://slack.openscope.co)
+[![Website Status](https://img.shields.io/website-up-down-green-red/http/openscope.co.svg)](http://www.openscope.co)
+
 ## Air Traffic Control
 
 An ATC simulator in HTML5.
 
 ## Running Locally
-It is possible to run this application locally.  You will need to have node and npm install in order for this to work.  For more information, view the [Tools Documentation](tools/README.md)
+
+It is possible to run this application locally. You will need to have [`node`](https://nodejs.org/en/download/) and [`npm`](https://docs.npmjs.com/getting-started/installing-node) installed in order for this to work. For more information, view the [tools documentation](tools/README.md).
 
 ## Privacy
 
-Page hits are tracked with Google Analytics; feel free to use an
-adblocker. The tracking is used only to get the number of page hits.
+Page hits are tracked with Google Analytics, so feel free to use an adblocker. The tracking is used only to get the number of page hits.
 
 ## Contributing
 
-Just create a pull request and I'll get to it. Thanks for helping!
+We use an [GitFlow Branching Model](http://nvie.com/posts/a-successful-git-branching-model) when adding features. If you would like to contribute, please [contact us](http://slack.openscope.co/) and we'll help as soon as we can! Thank you!
```

Comment:
Change this section to read:
_'We use the [GitFlow Branching Model](http://nvie.com/posts/a-successful-git-branching-model) for managing branches.  If you would like to contribute, you will be expected to use appropriate branch names based on this methodology (and we can help if you have questions)._

_Don't know Javascript?  Thats cool, we're always looking for beta testers and/or airport contributors.  If you would like to add a new airport, or help update existing airports, please read the [Airport Documentation]() to get up to speed on what is expected in that file.'_

In your airport documentation branch, be sure to add a link to the spawnPatterns doc (if you don't have it there already) because we moved it to a new folder in that branch.  It'll be easier to link from the airport file than from this file.

then add another section below that:


## Slack
Please join us on our [Slack Channel](http://slack.openscope.co/) if you have questions, comments or would like to contribute.  We can then add you to the organization so you can begin committing to this repo.